src/whidbey/repl.clj

```diff
@@ -0,0 +1,11 @@
+(ns whidbey.repl
+  (:require
+    [clojure.tools.nrepl.middleware.pr-values :refer [pr-values]]
+    [whidbey.render :as render]))
+
+
+(defn init!
+  "Initializes the repl to use Whidbey's customizations."
+  [options]
+  (render/update-options! options)
+  (alter-var-root #'pr-values (constantly identity)))
```

Comment:
Yeah, I'm trying to make sure any side-effecting stuff is only called from this function. That way you can import different parts of Whidbey to use without it necessarily modifying the environment.